vector/src/androidTest/java/im/vector/app/ui/robot/RoomDetailRobot.kt

```diff
@@ -103,7 +104,7 @@ class RoomDetailRobot {
 
     private fun longClickReaction(quickReaction: String) {
         withRetry {
-            longClickOn(quickReaction)
+            onView(withText(quickReaction)).perform(longClick())
```

Comment:
weird that it fixes something here... Does it mean that we should not use `longClickOn` anymore?

Reply:
I don't know if this will fix the CI flakyness~ but we weren't using `longClickOn` anywhere else :smile: and the `longClick()` was working well for the other interactions (open the message menu and room details)